app/models/user.rb

```diff
@@ -6,11 +6,19 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
+  before_destroy { self.photo = nil } # remove photo file from file system
+
   has_many :shf_applications
 
   has_many :payments
   accepts_nested_attributes_for :payments
 
+  has_attached_file :photo, default_url: 'photo_unavailable.png'
```

Comment:
"photo" is not a good name for an attribute because it's too generic (like "text" or "number").  It does not communicate anything about the intent (the intended use) for the attribute.  
I'd suggest "member_photo".
(Imagine in the future if there is a 2nd photo associated with a member.  Then this attribute name will definitely be ambiguous.)
I know this will be a PITA to change, but it'll be easier now than later.)

Reply:
Yes, the name is vanilla and useless.  I originally used `profile_photo` but dropped that since I don't believe we've established the _user profile_ as a real concept with the users yet (so using that term with client folks could be confusing).

I like your suggestion - will go with that.